packages/gatsby-remark-embed-snippet/README.md

```diff
@@ -86,7 +224,7 @@ html {
 }
 ```
 
-#### HTML example
+**HTML example**:
 
 ```html
 <html>
```

Comment:
For some reason I can't comment on lines 231-238...but in previewing this file, line 233 with `<!-- highlight-line -->` doesn't look like it would apply to anything. While you're in here, can you make an adjustment to move that highlight comment onto a line with code in it? e.g. `<p><!-- highlight-line -->`

Reply:
apparently husky pre-commit hook runs linters that is causing block comments to appear in a new line.

the only workaround i could get working is to add `packages/gatsby-remark-embed-snippet/README.md` to `.prettierignore`.